httpdocs/ajax/animation/list.php

```diff
@@ -43,17 +43,20 @@ function error($message) {
   $result = $mysqli->query($query) or error($mysqli->error);
   $animations = array();
   while($row = $result->fetch_array(MYSQLI_ASSOC)) {
-    settype($row['id'], 'integer');
-    settype($row['user'], 'integer');
-    settype($row['duration'], 'integer');
-    settype($row['size'], 'integer');
-    settype($row['viewed'], 'integer');
-    $row['title'] = htmlentities($row['title']);
-    $row['description'] = htmlentities($row['description']);
-    $row['version'] = htmlentities($row['version']);
-    $uri = '/' . $type . mysql_id_to_string($row['id']);
-    $row['url'] = 'https://' . $_SERVER['SERVER_NAME'] . $uri;
-    array_push($animations, $row);
+    if (!$row['uploading']) {
```

Comment:
I believe it is better to remove this test and to include an extra condition in the above `SELECT`, e.g., in line 21:
```php
  $extra_condition .= " AND uploading=0";
```